Drop `display` field

yinm opened the pull request:
resolve: https://github.com/tsconfig/bases/issues/180

orta reviewed (changes requested):
I'm open to this being post-processed out when we are going from the file in this repo to the file in the npm package, but not for removing the display tag inside these json files.

Having _display in ensures that each base json file is fully self contained

yinm commented:
Thank you for your review.

I changed code that drop `display` field when generating npm package.
Most of the codes are unusable, so I removed all commits and rebase latest main branch.

Does it accord that you think？

orta commented on bases/recommended.json:
```diff
@@ -1,12 +1,12 @@
 {
   "compilerOptions": {
-    "target": "es2016",
+    "target": "ES2015",
```
```suggestion
    "target": "ES2016",
```

yinm replied:
Fix 7ca1d5c...15ede8e

orta commented:
Seems reasonable to me 👍🏻

orta commented:
Thanks!
